### PR TITLE
Support SPN and SEN setup

### PR DESCRIPTION
**Pull request by 2dvorak**
This PR adds support for SPN and SEN installation and configuration.

Also, minor fix for the issue that ansible cannot be run multiple times.
That was because Klaytn not able to initialize due to already existing Klaytn chaindata.
So make ansible to wipe data directory before initializing Klaytn.

### CODE REVIEW

File: roles/klaytn_node/tasks/run_klaytn_tools.yml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  file:
+    path: "{{ homi_root }}/homi"
+    state: directory
+    mode: "0755"
+  when:
+    - not remoteNode|bool
+
+- name: Klaytn Homi Tool - Download Package
+  get_url:
+    url: "http://packages.klaytn.net/klaytn/{{ klaytn_homi.version }}/homi-{{klaytn_homi.version }}-0-linux-amd64.tar.gz"
```
Comment (henry-will) [inline roles/klaytn_node/tasks/run_klaytn_tools.yml:13]: In this case, Can you consider other os, too?

Comment (2dvorak) [reply roles/klaytn_node/tasks/run_klaytn_tools.yml:13]: Such change is being reviewed in #8.

**Comment by 2dvorak** [comment]
Rebased and force pushed to resolve conflict.

**Comment by 2dvorak** [comment]
Rebased and force pushed to resolve conflict.

File: roles/klaytn_node/tasks/run_klaytn_tools.yml [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    - ansible_os_family != "Darwin"
+
+- name: Klaytn kgen tool - Set kgen binary path
+  set_fact:
+    homi_bin: "{{ homi_root }}/kgen/kgen-darwin-10.10-amd64/bin/kgen"
+  tags: "launchKgen"
+  when:
+    - not remoteNode|bool
+    - ansible_os_family == "Darwin"
+
```
Comment (hyunsooda) [inline roles/klaytn_node/tasks/run_klaytn_tools.yml:194]: Duplicated newline may be removed

File: roles/klaytn_node/tasks/cp_genesis_files.yml [hunk 3]
```diff
@@ -1,32 +1,69 @@
 ---
-- name: Generate scn directories
+# Clear data directory if already exists
```
Comment (hyunsooda) [inline roles/klaytn_node/tasks/cp_genesis_files.yml]: Could you consider removing `klaytn_node_LOG_DIR` too?

**Comment by hyunsooda** [comment]
Please fix `NETWORK_ID` proeprty of `kpnd.conf` too (https://github.com/klaytn/klaytn-ansible/blob/master/roles/klaytn_node/templates/kpnd.conf.j2#L7)

File: roles/klaytn_node/tasks/run_klaytn_tools.yml (hunk 2 above)
Comment (2dvorak) [reply roles/klaytn_node/tasks/run_klaytn_tools.yml:194]: Done in 8d3fa2b. Marking as resolved

File: roles/klaytn_node/tasks/cp_genesis_files.yml (hunk 3 above)
Comment (2dvorak) [reply roles/klaytn_node/tasks/cp_genesis_files.yml]: Good point, thanks!! Done in 4890e41, marking as reesolved.

**Comment by 2dvorak** [comment]
> Please fix `NETWORK_ID` proeprty of `kpnd.conf` too (https://github.com/klaytn/klaytn-ansible/blob/master/roles/klaytn_node/templates/kpnd.conf.j2#L7)

@hyunsooda #13 has fix for that issue. Please refer to af85f22.

**Comment by hyunsooda** [comment]
Could you fix typos directly in this PR?
https://github.com/klaytn/klaytn-ansible/blob/3afa02136ef40ede19ec2895c5cc5ef0b7b60620/roles/klaytn_node/tasks/initialization.yml#L24,
https://github.com/klaytn/klaytn-ansible/blob/3afa02136ef40ede19ec2895c5cc5ef0b7b60620/roles/klaytn_node/tasks/initialization.yml#L18

**Comment by 2dvorak** [comment]
> Could you fix typos directly in this PR?
> 
> https://github.com/klaytn/klaytn-ansible/blob/3afa02136ef40ede19ec2895c5cc5ef0b7b60620/roles/klaytn_node/tasks/initialization.yml#L24
> 
> ,
> https://github.com/klaytn/klaytn-ansible/blob/3afa02136ef40ede19ec2895c5cc5ef0b7b60620/roles/klaytn_node/tasks/initialization.yml#L18

@hyunsooda Done, thanks!

**Review by hyunsooda — APPROVED** [review]

**Pull request merged**